tensorboard/plugins/mesh/mesh_plugin_test.py

```diff
@@ -209,7 +220,9 @@ def testsEventsAlwaysSortedByWallTime(self):
       "PluginRunToTagToContent",
       return_value={"bar": {"foo": "".encode("utf-8")}},
   )
-  def testMetadataComputedOnce(self, run_to_tag_mock):
+  @mock.patch.object(metadata, "get_current_version", return_value=0)
```

Comment:
Please don’t patch the `get_current_version` return value here. IIUC,
fixing `parse_plugin_metadata` to properly handle old data should
obviate this line for free. If for some reason that is insufficient, the
right fix wsould be to change the mocked return value on line 210 from
`"".encode("utf-8")` (which should really just be written `b""`, but no
matter) to a properly encoded proto with the appropriate contents.